### PR TITLE
Newcomers_Guide.rst: Rephrase note

### PR DESCRIPTION
**Pull request by mkfeuhrer**
**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [ ] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by mkfeuhrer** [comment]
I was not able to know that first thus i tried in mutiple commits . Also i ran coala locally but still gitmate error was there

**Comment by nemaniarjun** [comment]
@mkfeuhrer If you have any more doubts ping us on the gitter channel.
Marking this PR as a WIP because the tests are failing. :)

File: docs/Developers/Newcomers_Guide.rst [hunk 1]
```diff
@@ -263,8 +263,8 @@ For more information about reviewing code, check out this `link <http://coala.io
 
 .. note::
 
-    Reviewing code helps you by watching other people's mistakes and not making
-    them yourself in the future!
+    Reviewing code helps you by analyzing other people's
+    mistakes andthus you are improving yourself doing this!
```
Comment (RohanVB) [inline docs/Developers/Newcomers_Guide.rst]: `andthus` -> `and thus`
`yourself doing` -> `yourself by doing`

**Comment by RohanVB** [comment]
@mkfeuhrer just rebase and squash the extra commits, then push :)

**Comment by jayvdb** [comment]
fork deleted. cant be rescued.

**Pull request closed without merging**